### DISCUSSION TITLE
Create download link for the debriefing as PDF

### DISCUSSION THREAD

**Discussion by nrschoen** [jspsych/jsPsych#1657] [Q&A] [answered]
Hello,

Is there a way to let participants download a PDF of the debriefing? I guess basically I would need to create a download link for one of the files in the stimulus folder (being the debriefing).

Thanks in advance for your help!

**Comment by jodeleeuw** [comment, marked as answer]
You can use the [download attribute in an `<a>` tag](https://www.w3schools.com/tags/att_a_download.asp). I'd recommend combining this with something like the `html-button-response` plugin to show the link.

**Reply by nrschoen** [reply]
yup that does it! Thanks.
